src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import cn from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  error: ErrorType;
+  closeErrorMassege: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = React.memo(({
+  error,
+  closeErrorMassege,
+}) => (
+  <div
+    data-cy="ErrorNotification"
+    className={cn(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: error === ErrorType.NONE },
+    )}
+  >
+    {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+    <button
+      data-cy="HideErrorButton"
+      type="button"
+      className="delete"
+      onClick={closeErrorMassege}
+    />
+
+    {error === ErrorType.EMPTYTITLE && (
+      'Title can\'t be empty'
+    )}
+
+    {error === ErrorType.LOADING && (
+      'Unable to load todos'
+    )}
+
+    {error === ErrorType.ADD && (
+      'Unable to add a todo'
+    )}
+
+    {error === ErrorType.DELETE && (
+      'Unable to delete a todo'
+    )}
+
+    {error === ErrorType.UPDATE && (
+      'Unable to update a todo'
+    )}
```

Comment:
would be great to save somewhere error text and just use it here, or create separate component for errors with this logic